cmake/fw_zip.cmake

```diff
@@ -0,0 +1,36 @@
+
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+function(generate_dfu_zip zip_out bin_file_list deps)
```

Comment:
Could this go in extensions.cmake instead?

Reply:
No, we don't want patches in upstream extensions.cmake. 

Once we have an extensions.cmake in nrf repo we can move it there.
For now I think it makes sense to keep it here

Reply:
Ok, I see. Alternatively you could create it like in https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/2070, then solve the conflict whenever one is merged. It's up to you.